### PR TITLE
Fix logos

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Fix these logos on https://docs.getdbt.com/docs/trusted-adapters: 

- Apache Spark
- Postgres

Inspired by PR #4752 . h/t to @ntphiep

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).

### CODE REVIEW

File: website/snippets/_adapters-trusted.md [hunk 1]
```diff
@@ -127,17 +138,6 @@
   padding: 0rem 0.75rem; display:inline-block'>dbt Core</div>"
     icon="snowflake"/>
 
-<Card
-    title="Spark"
```
Comment (nghi-ly) [inline website/snippets/_adapters-trusted.md:131]: did not delete this. updated Card's title to use full trademarked name "Apache Spark" and alphabetized accordingly in list

https://spark.apache.org/

**Review by mirnawong1 — APPROVED** [review]
lgtm @nghi-ly ! one small fyi, for some reason the preview displays 2 column cards, and i noticed your localhost displayed 4. not sure if it's just a preview thing since the code has 4? but just an FYI in case!

**Comment by mirnawong1** [comment]
Screenshot 2024-03-08 at 13 14 52

https://github.com/dbt-labs/docs.getdbt.com/pull/5050#pullrequestreview-1925516019

**Comment by mirnawong1** [comment]
preview: https://docs-getdbt-fgmwegyp3-dbt-labs.vercel.app/docs/trusted-adapters

**Pull request merged**